docs/guides/advanced-usage/How-to-activate-Ant-Media-Server-from-within-restricted-geo-locations.md

```diff
@@ -0,0 +1,105 @@
+Ant Media uses Google services to verify the license key and those services are banned in China and Hong Kong. 
+
+For this reason, it’s not possible to verify the Ant Media license key using Google services.
+
+In this article, I'll provide two options to get around these restrictions and explain them step by step.
+
+The two options are:
+
+- A proxy service that is provided free for Ant Media Enterprise users.
+- A self-hosted proxy server
+
+## 1. Free Proxy service for Enterprise Users
+
+Ant Media offers this to Enterprise users for free. To access the free service, simply send an email to support@antmedia.io to receive a username and password.
+
+After receiving a username and password, run the following command then restart the Ant Media Server.
+
+```
+echo "proxy.address=username:password@proxy.antmedia.io:80" >> /usr/local/antmedia/conf/red5.properties
+systemctl restart antmedia
+```
+
+That's it! You can now verify your license over a restricted network. 
+
+## 2. Self-Hosted Proxy Server
+
+We are going to use Squid as the proxy server. 
+
+To get started, first have Ubuntu 22.04 or Ubuntu 20.04 server installed in an unrestricted region.
+
+Once you've got a running Ubuntu installation, continue to install Squid proxy with the below steps. 
+
+#### 1. Installing Squid server 
+```
+apt update
+apt install squid apache2-utils -y
+```
+#### 2. Backup existing configuration.
+
+mv /etc/squid/squid.conf{,_bck}
+
+#### 3. Modify the squid configuration
+
+Open the squid.conf file located at /etc/squid/squid.conf with an editor
```

Comment:
It would be better is if this line follows some standards to improve code quality. For example: Open the `squid.conf` file located at `/etc/squid/squid.conf` with an editor.